packages/kit/src/runtime/client/client.js

```diff
@@ -1202,13 +1202,13 @@ export function create_client({ target, base, trailing_slash }) {
 
 		goto: (href, opts = {}) => {
 			// TODO remove for 1.0
-			if ('keepfocus' in opts) {
+			if (!'keepFocus' in opts && 'keepfocus' in opts) {
 				throw new Error(
 					'`keepfocus` has been renamed to `keepFocus` (note the difference in casing)'
 				);
 			}
 
-			if ('noscroll' in opts) {
+			if (!'noScroll' in opts && 'noscroll' in opts) {
```

Comment:
```suggestion
			if ('noscroll' in opts && !('noScroll' in opts)) {
```

Reply:
Yeah i was already fixing this (while also screwing line endings again) -.-